clang/test/CodeGen/varargs-with-nonzero-default-address-space.c

```diff
@@ -0,0 +1,22 @@
+// REQUIRES: spirv-registered-target
```

Comment:
Does it actually? Normally -emit-llvm works regardless of the enabled backends

Reply:
I was not certain and erred on the side of caution. You are correct (as far as I can tell), and have removed the requirement. Thank you for pointing it out!